services/wallet/database.go

```diff
@@ -111,6 +96,10 @@ func (blob *JSONBlob) Value() (driver.Value, error) {
 	return json.Marshal(blob.data)
 }
 
+func NewDB(db *sql.DB) *Database {
+	return &Database{db}
```

Comment:
With this PR we are vulnerable to the problem described in https://github.com/status-im/status-go/issues/1537?

Reply:
yes that part wasn't added yet

Reply:
@adambabik made a separate pr to resolve that problem (https://github.com/status-im/status-go/issues/1537)